### PR TITLE
Intake

### PR DESCRIPTION
**Pull request by Arnav814**
## Overview

Some simulation stuff for the intake, and a command to intake a note.

### CODE REVIEW

File: src/main/java/frc/robot/commands/gpm/IntakeNote.java
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/commands/gpm/IntakeNote.java:1]: There are 2 IntakeNote commands. Delete one of them.

File: src/main/java/frc/robot/controls/GameControllerDriverConfig.java [hunk 1]
```diff
@@ -52,6 +52,10 @@ public void configureControls() {
                 : VisionConstants.RED_PODIUM_POSE,
             getDrivetrain()));
 
+    
+    //test for intake code
+    
+
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/controls/GameControllerDriverConfig.java:58]: Delete this comment and extra spaces.

File: src/main/java/frc/robot/controls/Operater.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return MathUtils.calculateHypotenuse(kDriver.get(Axis.RIGHT_X), kDriver.get(Axis.RIGHT_Y));
+  }
+
+  public boolean getIsSlowMode() {
+    return kDriver.RIGHT_TRIGGER_BUTTON.getAsBoolean();
+  }
+
+  public boolean getIsAlign() {
+    return kDriver.LEFT_TRIGGER_BUTTON.getAsBoolean();
+  }
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/controls/Operater.java:60]: Remove slow mode, is align, and anything else that isn't needed on operator. Also rename the controller.

**Comment by Ash1R** [comment]
@Arnav814 One small fix, also after this is merged we can make the entire Intaking note sequence (Arm in position + Intake Note).

File: src/main/java/frc/robot/subsystems/gpm/StorageIndex.java [hunk 3]
```diff
@@ -27,11 +27,14 @@ public class StorageIndex extends SubsystemBase {
...
    */
   public StorageIndex() {
-    m_indexmotor = new CANSparkFlex(StorageIndexConstants.indexMotorID, MotorType.kBrushless);
+    // m_indexmotor = new CANSparkFlex(StorageIndexConstants.indexMotorID, MotorType.kBrushless);***
+    m_indexmotor = new TalonFX(2);
     m_indexBeamBreak = new DigitalInput(StorageIndexConstants.indexBeamBreak);
     m_indexmotor.setInverted(false);
 
-    m_indexmotor.setIdleMode(StorageIndexConstants.idleMode);
+    // m_indexmotor.setIdleMode(StorageIndexConstants.idleMode); ***
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/gpm/StorageIndex.java:35]: Why is this commented out.

**Review by Kyle-Eldridge — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/gpm/IntakeNote.java
Comment (GLRoylance) [reply src/main/java/frc/robot/commands/gpm/IntakeNote.java:1]: We are merging 3 versions of the intake subsystem. The commands are somebody else's issue and can coexist for lnow.

File: src/main/java/frc/robot/controls/GameControllerDriverConfig.java (hunk 1 above)
Comment (GLRoylance) [reply src/main/java/frc/robot/controls/GameControllerDriverConfig.java:58]: The comment does not hurt and it is being worked on by someone else. The focus here is to get the intake subsystem merged into main. Don't demand perfection.

File: src/main/java/frc/robot/controls/Operater.java (hunk 2 above)
Comment (GLRoylance) [reply src/main/java/frc/robot/controls/Operater.java]: This does not hurt either. We need a fast merge.

File: src/main/java/frc/robot/subsystems/gpm/StorageIndex.java (hunk 3 above)
Comment (GLRoylance) [reply src/main/java/frc/robot/subsystems/gpm/StorageIndex.java:35]: Because someone commented it out. StorageIndex is in flux. Don't hold up a merge.

**Review by chem7397 — APPROVED** [review]

**Review by Kyle-Eldridge — APPROVED** [review]

**Pull request merged**